### ISSUE TITLE
[opencv] Error installing OpenCV

### ISSUE THREAD

**Issue by davidvelascogarcia** [roboticslab-uc3m/installation-guides#74]
Trying to install `opencv` in `ubuntu 14.04` PAL distro multiple errors appear doing `cmake ..` in build dir. Inspecting `CMakeError.log`, it shows that `No such file or directory
 <ffmpeg/avformat.h>, <sys/videoio.h>, <linux/videodev.h>`. However using the command `locate` headers appear in `usr/include`.
Also with `cmake ..` this error appear in loop before `Configuring incomplete, errors ocurred!`
```bash

CMake Warning (dev) at cmake/OpenCVPCHSupport.cmake:80 (GET_DIRECTORY_PROPERTY):
  Policy CMP0059 is not set: Do no treat DEFINITIONS as a built-in directory
  property.  Run "cmake --help-policy CMP0059" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.
Call Stack (most recent call first):
  cmake/OpenCVPCHSupport.cmake:256 (_PCH_GET_COMPILE_FLAGS)
  cmake/OpenCVPCHSupport.cmake:369 (add_precompiled_header)
  cmake/OpenCVModule.cmake:835 (ocv_add_precompiled_header_to_target)
  cmake/OpenCVModule.cmake:806 (_ocv_add_precompiled_headers)
  cmake/OpenCVModule.cmake:688 (_ocv_create_module)
  cmake/OpenCVModule.cmake:857 (ocv_create_module)
  modules/stitching/CMakeLists.txt:7 (ocv_define_module)
This warning is for project developers.  Use -Wno-dev to suppress it.
```
[Related](https://stackoverflow.com/questions/5842235/linux-videodev-h-no-such-file-or-directory-opencv-on-ubuntu-11-04)

Thanks

**Comment by davidvelascogarcia** [comment]
Configuring with `ccmake..`
```bash
 CMake Error at cmake/OpenCVDetectCXXCompiler.cmake:89 (math):
   math cannot parse the expression: "*100 + ": syntax error, unexpected
   exp_TIMES, expecting exp_OPENPARENT or exp_NUMBER (1)
 Call Stack (most recent call first):
   CMakeLists.txt:109 (include)
```

**Comment by davidvelascogarcia** [comment]
Solved changing `gcc` and `g++` version 7 to 4.9.
[Related](https://askubuntu.com/questions/26498/how-to-choose-the-default-gcc-and-g-version).

**Comment by davidvelascogarcia** [comment]
Closing issue.
